datasource/src/main/scala/astraea/spark/rasterframes/datasource/geotiff/GeoTiffInfoSupport.scala

```diff
@@ -46,7 +46,7 @@ trait GeoTiffInfoSupport {
   }
 
   def extractGeoTiffLayout(reader: ByteReader): (GeoTiffReader.GeoTiffInfo, TileLayerMetadata[SpatialKey]) = {
-    val info = GeoTiffReader.readGeoTiffInfo(reader, false, true)
+    val info = GeoTiffReader.readGeoTiffInfo(reader, false, true, None)
```

Comment:
Well at least because of this line its not compatible with 1.2

Reply:
Oh, that's right. Thanks for pointing that out. @metasim Looks like these changes aren't compatible then.

Reply:
Yea, that's because there were not `overviews` support in 1.2

Reply:
We're not ready to upgrade our stack to GT 2.x, so I may need to release separate versions, or figure out a reflections-based approach to handling both cases.

Next week I'll merge this PR and then work on that so we can run against either version.

@pomadchin Is there any chance in 2.x to provide backwards compatibility, or have the semantics changed greatly?

Reply:
If it would help you / us / @jbouffard  we can add an extra overload.

Reply:
Yeah, that's something we could do if it's needed.